### DISCUSSION TITLE
How to split the window in two?

### DISCUSSION THREAD

**Discussion by fayfive** [fyne-io/fyne#3240] [Q&A] [answered]
I want to divide the window to 2 parts and keeping One is 10% and the other is 90% and vertical

**Comment by shumichenko** [comment]
At first you need to know your window height, so you can calculate height of your blocks with this formula:
upperBlockHeight := windowHeight * 0.1
lowerBlockHeight := windowHeight - upperBlockHeight
To resize blocks you can create your custom layout or use GridWrapLayout

**Comment by Jacalz** [comment, marked as answer]
I think the `Split` container (https://pkg.go.dev/fyne.io/fyne/v2/container#Split) might be kind of what you are looking for.

**Reply by fayfive** [reply]
Spilt is ok. But I thank it's better to have a option similarly `enable_change_offset` to fixed the scale

**Reply by Jacalz** [reply]
I’m not sure what you mean. Would you mind clarifying?

**Reply by fayfive** [reply]
That means I `splite.SetOffset(0.01)`. The user can't change the offset in the UI.

**Reply by andydotxyz** [reply]
Why is it desirable to disallow user to control their user interface?

**Reply by fayfive** [reply]
Because I just want to split the window to two parts. One is 0.1 and the other is 0.9.
The 0.1 is similarly a title and the 0.9 is the content. The 0.9 has few characters

**Reply by shumichenko** [reply]
Then try to use a custom layout

**Reply by andydotxyz** [reply]
> Because I just want to split the window to two parts. One is 0.1 and the other is 0.9.
> 
> The 0.1 is similarly a title and the 0.9 is the content. The 0.9 has few characters

I don't really know what you are working on, but it sounds like formatted block of text would work, `NewRichTextFromMarkdown` is a great helper
